.github/pull_request_template.md

```diff
@@ -35,8 +46,8 @@ E.g.
 -->
 
 1.
-1.
-1.
+2.
+3.
```

Comment:
Just FYI that using consecutive `1`s in Markdown will automatically convert into an ordered list when displayed.

Reply:
Fixed!